HW_10.js

```diff
@@ -0,0 +1,130 @@
+// // Вивести на сторінку в один рядок через кому числа від 10 до 20.
+
+   let arr = [10, 12, 13, 14, 15, 15, 16, 17, 18, 19, 20];
+   for (let i = 0; i <= arr.length; i++); {
+       console.log(arr)
+
+   }
+
+
+// // Вивести квадрати чисел від 10 до 20.
+     
+     let  a1 = 0;
+    for ( let i = 10; i <=20; i++) {
+        a1 = i*i;
+        console.log(a1)
+    }
+
+
+// // Вивести таблицю множення на 7.
+
+
+    let num = '';
+	for (let i = 1; i <=7; i++) {
+		for (let j = 1; j <=7; j++) {
+			num  += (j*i) + "  ";       
+		}
+    console.log(i + " x");
+    console.log(num);
+		num = '';
+	} 
```

Comment:
Подвійний цикл у цьому заваданні -- це дуже важко і не підходить. 
Таблиця множиння на 7 це від 1 до 10. 

```suggestion
	for (let i = 1; i <=10; i++) {
	   console.log(i * 7)
	} 
```